leaderboards/static/html/grade_leaderboard_studio_edit.html

```diff
@@ -0,0 +1,37 @@
+{% load i18n %}
+<div class="wrapper-comp-settings is-active editor-with-buttons" id="settings-tab">
+    <ul class="list-input settings-list">
+        <li class="field comp-setting-entry is-set">
+            <div class="wrapper-comp-setting">
+                <label class="label setting-label" for="graded_target_id">Graded Component</label>
+                <select name="graded_target_id" id="graded_target_id">
+                    <option value="" {% if not graded_target_id %}selected="selected"{% endif %}>{% trans "Not Set" %}</option>
+                    {% for block in block_tree %}
+                        <option value="{{ block.id }}"{% if not block.eligible %} disabled="disabled"{% endif %}{% if block.is_this %} style="font-weight: bold;"{% endif %} {% if block.id == graded_target_id %}selected="selected"{% endif %} one="{{block.id}}" two="{{graded_target_id}}">
```

Comment:
Could this be broken into multiple lines so the logic can be better followed? The spacing shouldn't matter when parsing XML attributes. It may render a little weird, but we fix things by looking at the compiled DOM in the debugger, not view-source which isn't going to show us the content of the XBlocks anyway, since they're fetched via AJAX in most cases.